recipes/default.rb

```diff
@@ -19,8 +19,10 @@
 
 influxdb_config node['influxdb']['config_file_path'] do
   config node['influxdb']['config']
+  notifies :restart, 'service[influxdb]', :delayed
 end
 
 service 'influxdb' do
-  action [:enable, :start]
```

Comment:
this line is making kitchen tests fail.

```
       Recipe: influxdb-test::default
         * influxdb_database[test_database] action createE, [2017-02-22T01:02:14.764101 #268] ERROR -- InfluxDB: Failed to contact host localhost: #<Errno::ECONNREFUSED: Connection refused - connect(2) for "localhost" port 8086> - retrying in 0.01s.
       E, [2017-02-22T01:02:14.775251 #268] ERROR -- InfluxDB: Failed to contact host localhost: #<Errno::ECONNREFUSED: Connection refused - connect(2) for "localhost" port 8086> - retrying in 0.02s.
       E, [2017-02-22T01:02:14.795949 #268] ERROR -- InfluxDB: Failed to contact host localhost: #<Errno::ECONNREFUSED: Connection refused - connect(2) for "localhost" port 8086> - retrying in 0.04s.
       E, [2017-02-22T01:02:14.837212 #268] ERROR -- InfluxDB: Failed to contact host localhost: #<Errno::ECONNREFUSED: Connection refused - connect(2) for "localhost" port 8086> - retrying in 0.08s.
       E, [2017-02-22T01:02:14.919112 #268] ERROR -- InfluxDB: Failed to contact host 
```

Was there a reason why you removed it?

FYI, on initial setup of the cookbook, we require influxdb to be running so we can do some initialization of the database.   ref: https://github.com/bdangit/chef-influxdb/blob/master/test/fixtures/cookbooks/influxdb-test/recipes/default.rb#L5-L8

This is at least assuming the user needs to setup the db right after the service has been installed.